how-to/kubeflow_assets.md

```diff
@@ -0,0 +1,164 @@
+# Kubeflow Assets
+
+This doc is intended to provide an overview of the different assets 
+which are used by the Kubeflow community and identify the respective owners.
+
+## GitHub Org: kubeflow
+
+https://github.com/kubeflow
+
+The Kubeflow GitHub org is administered declaratively.
+
+Admins can be found in [org.yaml](https://github.com/kubeflow/internal-acls/blob/e4303ff3c7299bde05b4a9c7519e8592c5137755/github-orgs/kubeflow/org.yaml#L7)
+
+More info about how the GitHub org is administered in [GitHub Admin](https://github.com/kubeflow/community/blob/master/how-to/github_admin.md)
+
+
+### GitHub Robots
+
+https://github.com/kubeflow-bot
+
+* Password is stored in GCP Secret Manager project `kubeflow-admin`
+* You will also need a recovery token which are also stored in secret manager
+
+  * These are onetime use so remove it when you use it
+  * When you login you need to select use recovery token.
+
+## kubeflow.org GSuite Domain
+
+Admins
+
+* chrisheecho@kubeflow.org
+* ewj@kubeflow.org
+* jlewi@kubeflow.org
+* sarahmaddox@kubeflow.org
+* thea@kubeflow.org
+
+## Kubeflow GCP org
+
+This should be tied to the gsuite domain.
+
+Admins
+
+* jlewi@google.com
+* ricliu@google.com
+
+## DockerHub:
+
+https://hub.docker.com/u/kubeflow
+
+owners (docker ids):
+
+* carmark@
+* gaocege@
+* jlewigoogle@
+* k82cn@
+* seedjeffwan@
+
+### DockerHub Robot Accounts
+
+user: kubeflowrobot
+password:
+ 
+ * Secret dockerhub-kubeflowrobot in GCP Secret Manager In Project kubeflow-admin
+
+## Medium
+
+https://medium.com/kubeflow
+
+Editors
+
+* @chrisheecho_78982
+* @ankitbahuguna
+* @jlewi
+* @edd
+* @thealamkin
```

Comment:
```suggestion
* @thealamkin
* @animation2007
```

Reply:
I assume animation2007 is you? I don't have permission to add you as an editor; I think maybe Thea does. This doc is only intended to capture the current set of permissions. So I suggest we leave it as is and then you can follow up with Thea if you want to be added as an editor.

Reply:
sure